backends/src/lib.rs

```diff
@@ -338,10 +338,17 @@ async fn init_backend(
         {
             if let Some(api_repo) = api_repo.as_ref() {
                 let start = std::time::Instant::now();
-                download_onnx(api_repo)
+                let model_files = download_onnx(api_repo)
                     .await
-                    .map_err(|err| BackendError::WeightsNotFound(err.to_string()));
```

Comment:
Didn't you only need to add a `?` after the `map_err`?

Reply:
No because it still succeeds as far as I remember the `download_onnx` function was not producing errors but just warnings so empty ONNX files was not considered an error and so on it was printing that the files where there while those really weren't + this doesn't need to fail as it needs to fallback to the Safetensors backend if the ONNX files are not there.

Anyway I could've missed something, so please let me know!